src/main/java/org/commcare/formplayer/aspects/UserRestoreAspect.java

```diff
@@ -59,12 +61,21 @@ public void configureRestoreFactory(JoinPoint joinPoint) throws Throwable {
         HqAuth auth = getHqAuth((String) args[1]);
 
         configureRestoreFactory(requestBean, auth);
+        configureSentryScope(restoreFactory);
 
         if (requestBean.isMustRestore()) {
             restoreFactory.performTimedSync();
         }
     }
 
+    private void configureSentryScope(RestoreFactory restoreFactory) {
+        Sentry.configureScope(scope -> {
```

Comment:
I think I'm still a bit confused about this scope object.

Is Sentry managing this state per thread? Per Request?

Reply:
Responded in main comment